am-lib/src/main/java/uk/gov/hmcts/reform/amlib/repositories/AccessManagementRepository.java

```diff
@@ -1,13 +1,18 @@
 package uk.gov.hmcts.reform.amlib.repositories;
 
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
 import org.jdbi.v3.sqlobject.statement.SqlQuery;
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import uk.gov.hmcts.reform.amlib.models.AccessManagement;
 
 public interface AccessManagementRepository {
 
-    @SqlUpdate("insert into \"AccessManagement\" (\"resourceId\", \"accessorId\") values (?, ?)")
-    void createAccessManagementRecord(String resourceId, String accessorId);
+    @SqlUpdate("insert into \"AccessManagement\" (\"resourceId\", \"accessorId\", permissions) values (?, ?, ?)")
+    void createAccessManagementRecord(String resourceId, String accessorId, int permissions);
 
-    @SqlQuery("select exists(select 1 from \"AccessManagement\" where  \"accessorId\"=? and \"resourceId\"=?)")
-    boolean explicitAccessExist(String accessorId, String resourceId);
+    // The 'LIMIT 1' suffix was introduced because at the current database state (V2.2) there is a technical
+    // possibility of returning multiple records with this query, but it's forbidden from business point of view.
```

Comment:
This will be fixed soon. Maybe even in this sprint. We accept it for a very short while.

Reply:
(y)